.github/workflows/ci.yaml

```diff
@@ -113,6 +113,15 @@ jobs:
       - name: Running linting
         run: make lint
 
+  audit:
```

Comment:
My only question is: should we run this at the pull request triggering level or just after merges on master?

Reply:
I would run it for now a the PR level as w have not that many deps. Maybe later we can do it on master merge ? Let me know what you think.

Reply:
It's better to add it to the right place to avoid creating tech debt for the future. We can follow also what `fuel-core` has: https://github.com/FuelLabs/fuel-core/blob/master/.github/workflows/nightly-cargo-audit.yml and it as a separate Github action.

Reply:
@pedronauck  I have added the nightly-cargo-audit similarly to fuel-core. Now I have removed the audit step from the ci.yaml